app/interactors/restore_authorization_request_to_latest_authorization.rb

```diff
@@ -2,12 +2,29 @@ class RestoreAuthorizationRequestToLatestAuthorization < ApplicationInteractor
   def call
     return if latest_authorization.nil?
 
-    authorization_request.data = latest_authorization.data
+    restore_attributes!
+
     authorization_request.save
   end
 
   private
 
+  def restore_attributes!
```

Comment:
```suggestion
  def fill_authorization_request_with_latest_authorization_data!
```

Un peu verbeux, mais plus clair je trouve.

Reply:
Ça aurait été bien que tu fasses les 2 suggestions d'un coup pour que je commit directement depuis l'UI ;p (je vais le faire).

Reply:
ah oui j'ai pas pensé à renommer l'usage de la méthode x)